backends/tfhe-cuda-backend/build.rs

```diff
@@ -2,6 +2,10 @@ use std::env;
 use std::process::Command;
 
 fn main() {
+    if env::var("RUSTDOC").is_ok() {
+        return;
+    }
```

Comment:
Hmm it seems this breaks the build with tfhe-cuda-backend?

Reply:
yes that's my bad